src/mudlet.h

```diff
@@ -743,9 +745,6 @@ class translation
     // cases the loaded file will be a "xx" language only file even though it
     // is an "xx_YY" one here:
     QString mQtTranslationFileName;
-    // Further items like the above pair may be needed should some of the
```

Comment:
Don't remove this comment - it is entirely true and shouldn't be forgotten - the few translations in `qtkeychain` are ones that we need to get around to including in the near future - the author of that library has just added a Russian (ru_RU) translation in the 0.10 release BTW - see https://github.com/frankosterfeld/qtkeychain/tree/master/translations .

That is one aspect I intended when I made the `translation` class - so as to hold ALL the filenames we need to load to handle a particular GUI language (and unload them if/when that was changed)!

Reply:
I don't understand what this comment means, not sure anyone else does either. Can you reword it?

Reply:
We (will need) to include any bunch of `.ts` files that any subproject provides and that will need a `QString` in this class to hold the path to the selected file needed for the selected GUI language.

Specifically right now, someone (likely you or I) will need to add in the QKeyChain translations by:
* writing a `(void) mudlet::scanForQKeyChainTranslations(const QString& path)` which will be like `void mudlet::scanForQtTranslations(const QString& path)` except that it will look for a:
```CPP
    QString translationFileName(QStringLiteral("qkeychain_%1.qm").arg(languageCode));
```
to populate a `(QString) Translation::mQKeyChainTranslationFileName` member with the file that best matches the translations in `./3rdparty/qkeychains/translations`
* extend `(void) mudlet::loadTranslators(const QString&)` to also load that file after the Qt one and before the mudlet one...

As it happens I allowed for the need for further translation files to be wanted and any additional ones will automatically be handled provided we do the above for any subproject that includes them (as Qt `.ts` files).

Reply:
Okay - I've added it back with more understandable (for all) wording!